### DISCUSSION TITLE
Can I propagate source propagations?

### DISCUSSION THREAD

**Discussion by david-morris** [WeblateOrg/weblate#7120] [Q&A] [answered]
I would like to share certain terms between different components.

I have enabled translation propagation in my glossary and another component.  I then made a translation with a key "glossary.key" in both that component and my glossary.  I can change the translation of the key in the glossary, but when I download the other component's translation files, glossary.key retains its old value.

What am I doing wrong here?  I haven't been able to find any documentation describing how I can use translation propagation to share a translation between two components.  Only https://docs.weblate.org/en/latest/admin/projects.html and https://docs.weblate.org/en/latest/admin/continuous.html , but those don't describe propagating source strings. 
 Does translation propagation not work in source languages, or when the source language of components don't line up?
 
 Basically, is it possible to propagate by key rather than source string?

**Comment by nijel** [comment, marked as answer]
The propagation only happens for same source and key (or context).
